backend/tracim_backend/applications/content_file/controller.py

```diff
@@ -385,7 +394,9 @@ def preview_pdf_full_revision(self, context, request: TracimRequest, hapic_data=
         content = api.get_one(hapic_data.path.content_id, content_type=ContentTypeSlug.ANY.value)
```

Comment:
Do you have access to the content file extension from the API? If so, you could event avoid the api.get_one()